examples/pulse_control/4_Bell_pair_with_pulses_Rigetti.ipynb

```diff
@@ -0,0 +1,539 @@
+{
```

Comment:
<p>1) Lets specify what we mean by "all this data". This can be unclear to customers</p><p> </p><p>2) we <strong>can</strong> <s>call also</s> plot the waveform and see that it is <strong>a</strong> rounded square pulse. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>